nemo/export/tensorrt_llm.py

```diff
@@ -353,17 +366,38 @@ def export(
                     from megatron.core.export.trtllm.trtllm_helper import TRTLLMHelper
                     from tensorrt_llm.layers import MoeConfig
 
+                    use_embedding_sharing = model_configs.get("share_embeddings_and_output_weights", False)
+                    fp8_quantized, fp8_kvcache = determine_quantization_settings(
+                        model_configs, fp8_quantized, fp8_kvcache
```

Comment:
It's `model_configs` (plural) passed, but only 1x `nemo_model_config` consumed in `determine_quantization_settings` - could we unify?